### ISSUE TITLE
upgrade segments

### ISSUE THREAD

**Issue by MusTon79** [mautic/mautic#8047]
[//]: # ( Invisible comment: 
IIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIII
Before you create the issue:
IIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIII
Search for similar report among other reported issues.
Learn how to troubleshoot at https://www.mautic.org/docs/en/tips/troubleshooting.html
Use drag&drop to attach images or other files )

## Bug Description
I have the feeling that the segments do not update regularly
I DON'T UNDERSTAND IF THEY'RE GLOSING OR IT'S A BUT OF MAUTIC
Greetings.

| Q   | A
| --- | ---
| Mautic version | Last
| PHP version | PHP Version 7.0.33
| Browser | Chrome

### Steps to reproduce
1. I create a segment.
2. I apply a filter to the segment.
3. The segment is regularly populated.
4. I delete the filter in the segment.
5. The segment does not update regularly.
 
### Log errors
no one I believe

[//]: # ( Invisible comment:
Please check for related errors in the latest log file in [mautic root]/app/log/ and/or the web server's logs and post them here. Be sure to remove sensitive information if applicable. )

[2019-10-31 07-59-42.zip](https://github.com/mautic/mautic/files/3792515/2019-10-31.07-59-42.zip)

**Comment by npracht** [comment]
Hi @MusTon79 What is the latest version ? Also, i am scared to do not be able to reproduce with that description. Are you able to prove that there is an issue ? Do you find any of your contacts that are in or out and should be out or in ?

**Comment by MusTon79** [comment]
Hello. Thanks for the reply.

I wasn't able to verify the installed version but it was the last update available.

Having said that, I tell you that we downgraded to php version 5.6 and Mautic version 2.14.2.

The problem persists.

Have you looked at the attached file? I have attached a video showing the problem.

I also saw other people who have the same problem as me:
- https://github.com/mautic/mautic/issues/7679
- https://github.com/mautic/mautic/issues/5123

Everything is set up correctly. Cron and everything.

Thanks
